bing-dict.el

```diff
@@ -133,8 +133,52 @@ The value could be `synonym', `antonym', `both', or nil.")
                                          'face
                                          'font-lock-builtin-face))
 
+(defvar bing-dict--org-file "~/.orgs/vocabulary.org"
+  "The file where store the vocabulary.")
+
+(defvar bing-dict--org-file-title "Vocabulary"
```

Comment:
Users should be able to set this variable and the following variable, thus the names of these variables should not contain `--`. Use the single dash instead.